src/reactive/value.rs

```diff
@@ -3675,15 +3685,16 @@ impl Validations {
     /// The validation is linked with `self` such that checking `self`'s
     /// validation status will include this validation.
     #[must_use]
-    pub fn validate<T, E, Valid>(
+    pub fn validate<T, Valid, R>(
```

Comment:
Why was this changed from E to R? E stands for Error, I don't know what `R` would stand for.